lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -1600,6 +1600,9 @@ DefPrimitive('\@@@fontencoding{}', sub {
 DefMacroI('\f@encoding',  undef, sub { ExplodeText(LookupValue('font')->getEncoding); });
 DefMacroI('\cf@encoding', undef, sub { ExplodeText(LookupValue('font')->getEncoding); });
 
+## Defined since we "expect" it; Empty since we know nothing about it.
+DeclareFontMap('U', []);
```

Comment:
I had seem some missing_file warnings in arXiv with a small single letter `u`, I wonder if they're related?
https://corpora.mathweb.org/corpus/arxmliv/tex%5Fto%5Fhtml/warning/missing%5Ffile/u?all=false

Reply:
It may be worth an exploration; Previously if you typeset characters from a font with an "unknown" encoding and didn't otherwise end up with a fontmap, you'd get characters from whever the previous encoding was (ie. wrong chars). I think now you'll end up with them silently missing (which I'm inclined to think is what TeX would do?)

Reply:
Ok, new batch of commits. Seems better to not define a fontmap for U (default to OT1 encoding), define fontmaps in their own files, and make `LoadFontMap` quieter by default. That seems to act a bit more TeX-like.  So, thanks for the disruptive complement! :>

Reply:
Happy to be of help, I suppose :>